### PR TITLE
Fix/colang 2/flow continuation prompt

### PR DESCRIPTION
**Pull request by schuellc-nvidia**
* Fixes the assigned prompt role.
* Also increases a unit test threshold since it sporadically still fails.

### CODE REVIEW

File: nemoguardrails/llm/prompts/openai-chatgpt.yml [hunk 1]
```diff
@@ -222,7 +222,7 @@ prompts:
         content: "This is the current conversation between the user and the bot:"
       - "{{ history | colang | to_messages_v2}}"
 
-      - type: assistant
+      - type: system
```
Comment (sklinglernv) [inline nemoguardrails/llm/prompts/openai-chatgpt.yml:225]: are any other prompt templates (for other models) affected as well? Or they were already updated in the past?

**Review by sklinglernv — COMMENTED** [review]
Looks good, not sure if other template needs to be updated as well?

File: nemoguardrails/llm/prompts/openai-chatgpt.yml (hunk 1 above)
Comment (schuellc-nvidia) [reply nemoguardrails/llm/prompts/openai-chatgpt.yml:225]: Currently, we only support openai and llama3 models with the new interface. So, these are the only ones so far.

Comment (sklinglernv) [reply nemoguardrails/llm/prompts/openai-chatgpt.yml:225]: OK got it.

**Review by sklinglernv — APPROVED** [review]
All good from my side.

**Pull request merged**